tools/templates/IotDeviceBindingTemplate/dotnet_new_iot-device-binding_csharp/_IotDeviceBinding.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
I believe we should remove this as we want the bindings to be .dlls that can be included in consuming apps instead.

Reply:
Agreed and will remove.